### PR TITLE
torch-manager: Add flashlightd support

### PR DESCRIPTION
**Pull request by r3vn**
This Pull Request add flashlightd support to torch-manager.
Since we can't manage flashlight brightness level (yet), torch slider visibility has been disabled.

### CODE REVIEW

**Comment by MrCyjaneK** [comment]
Works on OnePlus 6T :+1:

File: src/torch-manager.c [hunk 1]
```diff
@@ -69,12 +72,16 @@ apply_brightness (PhoshTorchManager *self)
   const char *icon_name;
 
   g_return_if_fail (PHOSH_IS_TORCH_MANAGER (self));
-  g_return_if_fail (G_UDEV_IS_DEVICE (self->udev_device));
+  if (!DROIDIAN_TORCH_DBUS_IS_DROIDIAN_TORCH (self->droid_proxy)){
+    g_return_if_fail (G_UDEV_IS_DEVICE (self->udev_device));
 
+    self->brightness = g_udev_device_get_sysfs_attr_as_int_uncached (self->udev_device,
+                                                                   "brightness");
```
Comment (g7) [inline src/torch-manager.c]: nitpick: indentation

File: src/torch-manager.c [hunk 2]
```diff
@@ -69,12 +72,16 @@ apply_brightness (PhoshTorchManager *self)
...
+  } else {
+    self->brightness = droidian_torch_dbus_droidian_torch_get_brightness(self->droid_proxy);
```
Comment (g7) [inline src/torch-manager.c]: nitpick: missing space before `(self->droid_proxy)`

File: src/torch-manager.c [hunk 3]
```diff
@@ -234,16 +301,23 @@ phosh_torch_manager_idle_init (PhoshManager *manager)
...
                                                              (GAsyncReadyCallback) on_proxy_new_for_bus_finish,
                                                              self);
+
+  droidian_torch_dbus_droidian_torch_proxy_new_for_bus (G_BUS_TYPE_SESSION,
+                                                             G_DBUS_PROXY_FLAGS_NONE,
+                                                             "org.droidian.Flashlightd",
+                                                             "/org/droidian/Flashlightd",
+                                                             self->cancel,
+                                                             (GAsyncReadyCallback) on_droid_proxy_new_for_bus_finish,
+                                                             self);
```
Comment (g7) [inline src/torch-manager.c]: nitpick: indentation

**Review by g7 — APPROVED** [review]
LGTM, some style related nitpicks (just to keep in line with what phosh itself uses) but otherwise :100:, great work :tada: 

It would be nice to disable the toggle altogether if the camera/flashlight is not available but I don't think there is an easy way to do that...

**Comment by MrCyjaneK** [comment]
As a side note: camera wont work with flashlight on.

**Comment by r3vn** [comment]
> As a side note: camera wont work with flashlight on.

interesting, on my device torch turn off itself when pinhole is running, maybe we can open an issue on [meta](https://github.com/droidian/meta/issues) or [flashlightd](https://github.com/droidian/flashlightd/issues) directly for this

**Pull request merged**